action/alarmWebAction.js

```diff
@@ -191,7 +191,7 @@ function main(params) {
                         reason: doc.status.reason
                     }
                 };
-                var strict = true; // strict is default to true
+                var strict = false; // strict is default to false
```

Comment:
I suppose this should be the default value from the deployment configuration rather than just one of true or false?

Reply:
Yes, you are right.
So i opened another pr: https://github.com/apache/openwhisk-package-alarms/pull/216